HISTORY.md

```diff
@@ -0,0 +1,13 @@
+0.0.3 / 2024-08-27
+------------------
+* Enable returning metadata by @hammannr in https://github.com/XENONnT/inference_interface/pull/7
+* Fix typos by @yuema137 in https://github.com/XENONnT/inference_interface/pull/10
+* Better metadata loading for toyfiles by @hammannr in https://github.com/XENONnT/inference_interface/pull/8
+* Fix hist1d interface by @yuema137 in https://github.com/XENONnT/inference_interface/pull/9
+* Better treatment of empty dicts by @hammannr in https://github.com/XENONnT/inference_interface/pull/11
+
+## New Contributors
```

Comment:
`## New Contributors` -> `New Contributors`

Reply:
you're of course right -- sorry

Reply:
fixed now